src/EducationService.Models.Db/DbCertificateImage.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using System;
+
+namespace LT.DigitalOffice.EducationService.Models.Db
+{
+  public class DbCertificateImage
+  {
+    public const string TableName = "CertificateImages";
+
+    public Guid Id { get; set; }
+    public Guid CertificateId { get; set; }
+    public Guid ImageId { get; set; }
+
```

Comment:
createdBy, createdAtUtc

Reply:
We usually do not put such fields in service tables.